src/main/java/slimeknights/tconstruct/library/recipe/worktable/ModifierSetWorktableRecipeBuilder.java

```diff
@@ -20,21 +20,21 @@
 @RequiredArgsConstructor(access = AccessLevel.PRIVATE)
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 public class ModifierSetWorktableRecipeBuilder extends AbstractSizedIngredientRecipeBuilder<ModifierSetWorktableRecipeBuilder> {
-  private final ResourceLocation dataKey;
+  private final ResourceLocation key;
```

Comment:
Same question here, why did this change?

Reply:
Yea. It was overzealous replace. I'm just remembering while typing it was far worst so you are seeing the two entries I missed while fixing.